src/CMakeLists.txt

```diff
@@ -345,20 +402,26 @@ if(CMAKE_CROSSCOMPILING)
   endforeach(firmware)
 
   target_sources(firmware.elf PRIVATE firmware.c)
-  target_compile_definitions(firmware.elf PRIVATE APP_BTC APP_LTC APP_ETH APP_U2F)
+  target_compile_definitions(firmware.elf PRIVATE FIRMWARE_MULTI APP_BTC APP_LTC APP_ETH APP_U2F)
   target_sources(firmware.elf PRIVATE ${FIRMWARE-U2F-SOURCES})
 
   target_sources(firmware-semihosting.elf PRIVATE firmware.c)
   target_sources(firmware-semihosting.elf PRIVATE ${FIRMWARE-U2F-SOURCES})
   # Select an implementation of the system calls that can communicate with the debugger
   target_compile_options(firmware-semihosting.elf PRIVATE --specs=rdimon.specs)
   target_link_libraries(firmware-semihosting.elf PRIVATE --specs=rdimon.specs)
-  target_compile_definitions(firmware-semihosting.elf PRIVATE APP_BTC APP_LTC APP_ETH APP_U2F)
+  target_compile_definitions(firmware-semihosting.elf PRIVATE FIRMWARE_MULTI APP_BTC APP_LTC APP_ETH APP_U2F)
   target_compile_definitions(firmware-semihosting.elf PRIVATE SEMIHOSTING)
 
   target_sources(firmware-btc.elf PRIVATE firmware.c)
   target_compile_definitions(firmware-btc.elf PRIVATE FIRMWARE_BTC_ONLY APP_BTC)
 
+  target_sources(firmware-base.elf PRIVATE base.c)
+  target_compile_definitions(firmware-base.elf PRIVATE FIRMWARE_BASE APP_BTC)
```

Comment:
`FIRMWARE_BITBOXBASE`.

reading `#ifdef FIRMWARE_BASE` is not descriptive enough

Reply:
ack